File: DSA/src/DataStructures/Graph.java
package DataStructures;
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

// Class Structure
public class Graph {
    public static void main(String[] args) {
        // Signature
        Map<Integer, List<Integer>> graph = new HashMap<>();

        // Add Vertex
        graph.put(1, new ArrayList<>());

        // Add Edge
        graph.get(1).add(2);

        // Remove Vertex
        graph.remove(1);

        // Remove Edge
        graph.get(1).remove(Integer.valueOf(2));
    }
}

File: DSA/Python/DataStructures/graph.py
def main():
    # Initialize the graph as a dictionary
    graph = {}

    # Add Vertex
    graph[1] = []

    # Add Edge
    graph[1].append(2)

    # Remove Vertex
    graph.pop(1, None)

    # Add Vertex again for removing edge demonstration
    graph[1] = [2, 3]

    # Remove Edge
    graph[1].remove(2)

    print("Graph:", graph)

if __name__ == "__main__":
    main()